Example/Podfile

```diff
@@ -14,7 +14,7 @@ target 'Emission' do
     pod 'Artsy+UIFonts', :git => "https://github.com/artsy/Artsy-OSSUIFonts.git"
   end
 
-  pod 'React', :path => react_path, :subspecs => ['RCTWebSocket']
+  pod 'React', :path => react_path, :subspecs => ['RCTWebSocket', 'RCTImage']
```

Comment:
We can use our own image view, but I kept getting this error with it: `Touchable child must either be native or forward setNativeProps to a native component` (https://facebook.github.io/react-native/docs/direct-manipulation.html), and when I tried wrapping it in a dummy view, the error went away but the resulting image was just a black square. I can try using the `setNativeProps` solution outline in the RN docs, but it seemed easier to just use the most basic `Image` component provided by RN.

Do you think using `setNativeProps` is a better way to go?